README.md

```diff
@@ -7,20 +7,18 @@ The library can be used by importing `github.com/projectdiscovery/cdncheck`. her
 package main
 
 import (
-    "log"
-    "net"
-    "github.com/projectdiscovery/cdncheck"
+	"log"
+	"net"
+	"github.com/projectdiscovery/cdncheck"
 )
 
 func main() {
-    client, err := cdncheck.New()
-    if err != nil {
-        log.Fatal(err)
-    }
-    if found, _, _, err := client.Check(net.ParseIP("173.245.48.12")); found && err == nil {
-        log.Println("ip is part of cdn")
-    }
+	client := cdncheck.New()
+	if found, _, _, err := client.Check(net.ParseIP("173.245.48.12")); found && err == nil {
+		log.Println("ip is part of cdn")
+	}
 }
+
```

Comment:
@RamanaReddy0M , I think we can add a better example add use below methods instead of generic `Check`
- `CheckCDN`
- `CheckWAF`
- `CheckCloud`

not exactly this but something similar
```go
        client := cdncheck.New()
	ip := net.ParseIP("173.245.48.12")
	matched, val, err := client.CheckCDN(ip)
	if err != nil {
		panic(err)
	}
	if matched {
		// found a match
		fmt.Printf("%v is a %v\n", ip, val)
	} else {
		// no match found
		fmt.Printf("%v is not a cdn\n", ip)
	}
```